src/Microsoft.AspNet.Mvc.TagHelpers/Internal/IGlobbingUrlBuilder.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
+{
+    public interface IGlobbingUrlBuilder
```

Comment:
The `GlobbingUrlBuilder's` `BuildUrlList` method is already virtual though, shouldn't need the interface.